react/week3/ToDoList.js

```diff
@@ -0,0 +1,148 @@
+import React, { useState, useEffect } from 'react';
+
+
+function ToDoList() {
+
+const URL = "https://gist.githubusercontent.com/benna100/391eee7a119b50bd2c5960ab51622532/raw";
+    
+useEffect(() => {
+  async function fetchData() {
+      const response = await fetch(URL)
+      const json = await response.json();
+    setMytodolist(json)
+  }
+  fetchData();
+}, []); 
+    
+    const [description, setDescription] = useState('');
+    const [deadline, setDeadline] = useState('');
+    const [mytodolist, setMytodolist] = useState([]);
+    const [checkedItem, setCheckedItem] = useState({ renderTrigger: true })
+    const [passedTime, setpassedTime] = useState(0);
+    
+    //ASK MENTOR: whithout useffect below function works as well. why should be use useEffect?
+
+    useEffect(() => {
```

Comment:
The idea was to keep it separate from the other component , so it will not have effect of changing the component in each second.